frontend/src/App.tsx

```diff
@@ -3,7 +3,7 @@ import Home from './pages/Home'
 const App = () => {
   return (
     <>
-    <Home />
+<Home></Home>
```

Comment:
I tag JSX sono generalmente in autoclose, quindi vanno scritti < nomecomponente /> e non così